File: java/src/main/java/challenges/aoc2023/day1/Application.java
package challenges.aoc2023.day1;

import challenges.aoc2023.day1.service.CalibrationService;
import challenges.aoc2023.day1.service.DayOneSolver;
import service.TaskSolver;

public class Application {
    public static void main(String[] args) {
        final TaskSolver solver = new TaskSolver(new DayOneSolver(new CalibrationService()));
        solver.solveTasks();
    }
}

File: scala/src/main/scala/challenges/aoc2025/day1/Application.scala
package challenges.aoc2025.day1

import _root_.service.TaskSolver
import challenges.aoc2025.day1.service.{DayOneSolver, SafeService}

object Application {
  def main(args: Array[String]): Unit = {
    val solver: TaskSolver = TaskSolver(DayOneSolver(SafeService()))
    solver.solveTasks()
  }
}
